### ISSUE TITLE
[Bug]: tModLoader fails to install

### ISSUE THREAD

**Issue by ChrisMathews393** [parkervcp/eggs#1850]
### Panel Version

1.10.1

### Wings Version

1.7.0

### Service

tmodloader

### Modified

No, I did not modify the egg

### Expected Behavior

Install the server and start

### Actual Behavior

Server does not start, fails to find the start script
Files tab is empty in ptero UI
Installer errors with  "unknown filetype. Exiting"

### Steps To Reproduce

Install tmodloader from egg

### Install logs

https://pteropaste.com/5jqast

**Comment by QuintenQVD0** [comment]
I think the egg can not install v1.4 (so latest)

**Comment by ChrisMathews393** [comment]
Is there a way to change the version that it's trying to pull without breaking everything? I'm sure I could edit the json but not sure if thats the best way

**Comment by QuintenQVD0** [comment]
> Is there a way to change the version that it's trying to pull without breaking everything? I'm sure I could edit the json but not sure if thats the best way

There is a version variable set it to a valid v 1.3 release tag

**Comment by ChrisMathews393** [comment]
Same result with "v0.11.7.2" in the version field

**Comment by ChrisMathews393** [comment]
Tried several versions, including editing the egg itself rather than startup variables on the server, same result.

**Comment by ChrisMathews393** [comment]
Figured it out, for some reason the docker image variable on the egg configuration was set to "debian mono" or something similar, and it needed to be set to .NET (I only discovered this by re-adding the egg without changing anything and having it work). I know for sure that the configuration was not modified by me, but not sure how else it would have broken. In any case, I guess that's what it was. Latest version pulls fine.
